spring-cloud-starter-tencent-polaris-router/src/main/java/com/tencent/cloud/polaris/router/feign/PolarisFeignLoadBalancer.java

```diff
@@ -70,18 +73,15 @@ PolarisRouterContext buildRouterContext(Map<String, Collection<String>> headers)
 		routerContext.setLabels(PolarisRouterContext.TRANSITIVE_LABELS, MetadataContextHolder.get()
 				.getFragmentContext(MetadataContext.FRAGMENT_TRANSITIVE));
 
-		labelHeaderValues.forEach(labelHeaderValue -> {
-			Map<String, String> labels = JacksonUtils.deserialize2Map(labelHeaderValue);
-			if (!CollectionUtils.isEmpty(labels)) {
-				Map<String, String> unescapeLabels = new HashMap<>(labels.size());
-				for (Map.Entry<String, String> entry : labels.entrySet()) {
-					String escapedKey = ExpressionLabelUtils.unescape(entry.getKey());
```

Comment:
delete `com.tencent.cloud.common.util.ExpressionLabelUtils#unescape` method.

Reply:
In 2020.0 and Greenwich branch?